brave-lists/brave-cookie-specific.txt

```diff
@@ -1,6 +1,7 @@
-! Cookie-snippet list
-! Used with cookie snippets, set-local-storage-item, set-cookie, set-cookie-reload, set-ession-storage-item
-!
+! Title: Brave Cookie-snippet list
+! Description: Used with cookie snippets, set-local-storage-item, set-cookie, set-cookie-reload, set-ession-storage-item
```

Comment:
I think better to describe it generally rather than listing specific scriptlet names - e.g. we'll likely move to uBO's `trusted-set-cookie` in the future

something like `Brave-specific additions to Easylist Cookie`

Reply:
Yeah, It crossed my mind that I could have removed the field as well, since only Social had a Description and Brave doesn't do anything with it either (at least for now).
But updated it with your suggestion, so it makes more sense now.
Thanks!.